index.html

```diff
@@ -16,11 +16,11 @@
       <section id="content">
         <form id="search-form" action="#" onsubmit="return false">
           <input
-            id="searchInput"
-            name="searchInput"
+            id="search-input"
+            name="search-input"
```

Comment:
For consistency with other class and ID names, the camelCase `searchInput` would be better as the kebab-case `search-input`.